src/main/resources/view/RedTheme.css

```diff
@@ -0,0 +1,204 @@
+.background {
+    -fx-background-color: #ff4745;
+}
+
+.background2 {
+    -fx-background-color: #f4f4f4;
+}
+
+.text-field,.text-area, .split-pane,.tab {
+	-fx-border-color: #2e4053;
```

Comment:
You are mixing tabs and spaces for indentation. The configuration we are given does not auto convert tabs to spaces, but there's no web tools platform on Eclipse for Java Developers so we can't do that either. You can either install it:
http://download.eclipse.org/webtools/updates/

Or just fix the indentation for the few lines since we won't be touching the css file much.

Although we are not told the style to follow for CSS, it's best to keep it consistent to use spaces for CSS as well. Furthermore, mixing spaces and tabs for indentation is not a good idea.

Since you are already using 4 spaces for most of your CSS file, convert all your tabs to spaces